src/pipelines_repository/adf-build/tests/stubs/stub_deployment_map.yml

```diff
@@ -27,3 +27,12 @@ pipelines:
       - NotificationEndpoint: my_email@email.com
     targets:
       - 123456789
+
+  - name: buildonly
+    type: cc-buildonly
+    params:
+      - SourceAccountId: 11111233321
+      - NotificationEndpoint: my_email@email.com
+    targets:
+      - regions: eu-west-1
```

Comment:
Currently we don't have the possibility to have the "build" phase in a different region other than the deployment region. I feel this would make more sense to just remove the "targets" key.

Reply:
Yes, i agree that would make the most sense but the problem is that the build stage wants write its output to a ArtifactStore thats currently being generated based on the regions provided in the target.